packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -322,6 +322,10 @@ export interface IOS {
    * The background color for your iOS app, behind any of your React views. Overrides the top-level `backgroundColor` key if it is present. Requires `expo-system-ui` be installed in your project to work on iOS.
    */
   backgroundColor?: string;
+  /**
+   * The Apple development team ID to use for all build configurations.
+   */
+  developmentTeam?: string;
```

Comment:
should we call this `teamID`? it seems to be more frequently referred to by apple as "Team ID" - eg: https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/

Reply:
Sure, I can change it. It's not really consistent from apple though: 
![image](https://github.com/user-attachments/assets/e737a843-0f80-4706-91ca-a609948b7548)

[source](https://developer.apple.com/help/account/manage-your-team/invite-team-members/)

Reply:
maybe `appleTeamId` for consistency with eas submit profiles? https://docs.expo.dev/submit/eas-json/#production-profile

Reply:
Let's go for `appleTeamId`, I'll update the XDL schema.